google-cloud-spanner-hibernate-samples/spanner-hibernate-codelab/README.adoc

```diff
@@ -0,0 +1,5 @@
+= Cloud Spanner Hibernate Codelab
+
+This is the solution for the Cloud Spanner with Hibernate ORM Codelab.
+
+Please refer to the codelab for instruction
```

Comment:
Add period at end of sentence.

Reply:
Also I would suggest circling back here and adding a link from the docs to the codelab once it is ready. You can create an issue as a reminder to do this.